Modules/Core/src/Services/CustomFieldsService.php

```
@@ -344,7 +346,7 @@ private function renameColumn($table_name, $old_column_name, $new_column_name)
      */
     private function addColumn($table_name, $column_name)
     {
-        $this->load->dbforge();
+// TODO: Use Laravel migrations - $this->load->dbforge();
         $column = [$column_name => ['type' => 'VARCHAR', 'constraint' => 256]];
         $this->dbforge->addColumn($table_name, $column);
     }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'addColumn'. (undefined)

(UnusedPrivateMethod)